model/src/cfd.rs

```diff
@@ -833,7 +836,11 @@ impl Cfd {
             bail!("Can only accept proposal as a maker");
         }
 
-        let hours_to_charge = self.hours_to_extend_in_rollover()?;
+        let hours_to_charge = match version {
+            Version::V1_0_4_7 => 1,
+            Version::Current => self.hours_to_extend_in_rollover()?,
+        };
```

Comment:
👍 Having discussed the merits of duplicating code vs introducing branching logic I think this is okay. We can always revisit this choice in the future if we notice that we're having to introduce a lot of code like this.